SECURITY.md

```diff
@@ -0,0 +1,11 @@
+# Security Policy
+
+## Supported Versions
+
+Security updates are applied only to the most recent releases.
+
+## Reporting a Vulnerability
+
+To securely report a vulnerability, please email to kataw@gmail.com.
```

Comment:
@KFlash what do you think leaving the email??

Reply:
we don't have that email adr

Reply:
your personal email?

Reply:
what about yours?